client/src/components/buttons/buttons.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+
+function Buttons(props) {
+  if (props.value === 'search') {
```

Comment:
This button will only render if the value prop is set to search.
It's better to set the value as whatever we give it from props, rather than only rendering a button when the props.value is equal to search.
```suggestion
  return <input className="button" type="submit" value={props.value} />;
```